lib/cli/commands/validate.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const fs = require('fs-extra')
+const path = require('path')
+const pathContains = require('contains-path')
+const Ajv = require('ajv')
+
+const supportSchema = require('./../../../schema.json')
+
+module.exports = function (opts) {
+  return {
+    command: 'validate',
+    desc: 'Validate support information schema',
+    builder: function (yargs) {
+      return yargs
```

Comment:
```suggestion
      return yargs;
```
but we can address this later ;-)

Reply:
`standard` doesn't validate the `;` at the end of line